### PR TITLE
#7832 Context menus not showing in mybinder

### PR DESCRIPTION
**Pull request by piorek**

### CODE REVIEW

File: js/notebook/src/plot/contextMenu/plotContextMenu.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default class PlotContextMenu extends BkoContextMenu {
       ...createSaveAsMenuItems(this.scope),
     ];
 
-    new BeakerXApi(`${window.location.origin}/`)
+    new BeakerXApi(`${(Jupyter.notebook_list || Jupyter.notebook).base_url}`)
```
Comment (LeeTZ) [inline js/notebook/src/plot/contextMenu/plotContextMenu.ts:33]: Could you explain why this change fixes the menu issue?

Comment (piorek) [reply js/notebook/src/plot/contextMenu/plotContextMenu.ts:33]: There was uncaught exception caused by 404 while requesting /settings endpoint (needed to see if publishing is enabled).

This one is for plots but is exactly the same case.
